### DISCUSSION TITLE
Uçaklar ve havaalanları sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#108] [Zeka Sorusu]
Bir ülkede bir çok hava alanı ve her hava alanında bir uçak vardır. Alarm verilince her hava alanındaki uçak havalanacak ve kendine en yakın hava alanına inecektir. Buna göre bir hava alanına en çok kaç uçak inebilir?

**Comment by tasgin123** [comment]
6. dairenin yarıçapı r ve cevresi de 2pi.r olursa dairenin merkezindeki bir havaalanı iöin (2.pi.r)/r = 2.pi sonucuna ulaşırız. bu da 6 ve 7 arasında bir sayı oldugu için cevap **_6_**
